_locales/it/messages.json

```diff
@@ -0,0 +1,300 @@
+{
+	"additionalPermissions": {
+		"message": "Permessi aggiuntivi"
+	},
+
+	"all": {
+		"message": "Globale"
+	},
+
+	"alwaysUsesHttp": {
+		"message": "Usa sempre http"
+	},
+
+	"alwaysUsesHttps": {
+		"message": "Usa sempre https"
+	},
+
+	"ask": {
+		"message": "Chiedi"
+	},
+
+	"bookmarks": {
+		"message": "leggere e modificare i segnalibri"
+	},
+
+	"default": {
+		"message": "Predefinito"
+	},
+
+	"enabled": {
+		"message": "Abilitata"
+	},
+
+	"errorHTMLTitle": {
+		"message": "Errore",
+		"description": "The title of error.html."
+	},
+
+	"extensionDescription": {
+		"message": "Apri un link nella barra laterale.",
+		"description": "Description of the extension."
+	},
+
+	"extensionName": {
+		"message": "Apri nella barra laterale",
+		"description": "Name of the extension."
+	},
+
+	"host": {
+		"message": "accedere ai dati dei siti web"
+	},
+
+	"iconImageAlt": {
+		"message": "L'icona di questa estensione.",
+		"description": "The alternate description of the icon of this extention."
+	},
+
+	"indexHTMLCaution": {
+		"message": "Nota: selezionando un url con protocollo https, sarà privilegiata l'opzione https anche se si sceglie http.",
+		"description": "Cautions of this extention."
+	},
```

Comment:
Google Translation says "Nota" means "Note". It seems not suitable for "Caution".